main/utils.zkasm

```diff
@@ -872,10 +872,13 @@ SHLarithfinal:
 
 ; out of counters full tracer event trigger
 outOfCountersStep:
+    $                           :MLOAD(isLoadRLP),JMPNZ(handleOOCSatRLP)
```

Comment:
What do you think adding this check only once at `handleBatchError` function?
**$                           :MLOAD(isLoadRLP),JMPNZ(handleOOCatRLP)**
And remove the `handleOOCXatRLP` from the code